pkg/route/ingress/ingress.go

```diff
@@ -439,13 +438,21 @@ func (c *Controller) sync(key queueKey) error {
 		if err != nil {
 			return err
 		}
-		data = []byte(fmt.Sprintf(`[{"op":"replace","path":"/spec","value":%s}]`, data))
+
+		if route.Annotations != nil {
+			annotations, err := json.Marshal(&route.Annotations)
+			if err != nil {
+				return err
+			}
+			data = []byte(fmt.Sprintf(`[{"op":"replace","path":"/spec","value":%s},`+`{"op":"replace","path":"/metadata/annotations","value":%s}]`, data, annotations))
```

Comment:
Might as well wrap lines where the string is broken:
```suggestion
			data = []byte(fmt.Sprintf(`[{"op":"replace","path":"/spec","value":%s},`+
			`{"op":"replace","path":"/metadata/annotations","value":%s}]`,
			data, annotations))
```